src/registerProviders.test.ts

```diff
@@ -0,0 +1,50 @@
+describe("Register Providers", () => {
+
+    beforeEach(() => {
+        jest.resetModules();
+    });
+
+    describe("Browser Registration", () => {
+        afterEach(() => {
+            process.env.TEST = "true";
+        });
+
+        it("Doesn't Register localFileSystemProxy", () => {
+            process.env.TEST = "false";
+
+            const mock = jest.mock("os", () => {
+                return {
+                    release: () => "browser",
+                };
+            });
+
+            const registerProviders = require("./registerProviders").default;
```

Comment:
We have standardized on using `import` vs. `require`.  Please update and move module imports to top of file.